frontend/src/pages/MetadataByType/MetadataByTypeResults.jsx

```diff
@@ -100,19 +94,25 @@ const MetadataByTypeResults = ({ metadataTypeUuid }) => {
   if (loading) {
     return (
       <CategoriesContainer>
-        <ResultsHeader>
-          <h1>{metadataTypeName}</h1>
-        </ResultsHeader>
         <NoResult text="Loading..." />
       </CategoriesContainer>
     );
   }
+  if (!metadataType) {
+    return (
+      <CategoriesContainer>
+        <NoResult text={`No category found for ${metadataTypeUuid}`} />
+      </CategoriesContainer>
+    );
+  }
+  const { tags, description, name } = metadataType;
   return (
     <CategoriesContainer>
       <ResultsHeader>
         <div>
-          <h3>{metadataTypeName}</h3>
-          <p>{description}</p>
+          <h3>{metadataType.name}</h3>
+          <DescriptionEditButton uuid={metadataTypeUuid} currentDescription={description.content} />
```

Comment:
DescriptionEditButton has no prop currentDescription

```suggestion
          <DescriptionEditButton uuid={metadataTypeUuid} />
```

Reply:
Good catch